### PR TITLE
add uds details to public documentation

### PR DESCRIPTION
**Pull request by krakjoe**
### What does this PR do?
Adds details of UDS support to public PHP documentation.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by bwoebi — APPROVED** [review]

File: content/en/tracing/trace_collection/library_config/php.md [hunk 1]
```diff
@@ -162,7 +162,17 @@ The Agent request transfer timeout (in milliseconds).
 `DD_TRACE_AGENT_URL`
 : **INI**: `datadog.trace.agent_url`<br>
 **Default**: `null`<br>
-The Agent URL; takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`; for example: `https://localhost:8126`. Added in version `0.47.1`.
+The Agent URL; takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`; for example: `https://localhost:8126`. Added in version `0.47.1`. Supports UDS (Unix Domain Sockets) via `unix://`.
```
Comment (kayayarai) [inline content/en/tracing/trace_collection/library_config/php.md:165]:
```suggestion
The Agent URL; takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`; for example: `https://localhost:8126`. Added in version `0.47.1`. Supports Unix Domain Sockets (UDS), using `unix://`.
```

File: content/en/tracing/trace_collection/library_config/php.md [hunk 2]
```diff
@@ -162,7 +162,17 @@ The Agent request transfer timeout (in milliseconds).
...
+
+`DD_DOGSTATSD_URL`
+: **INI**: `datadog.dogstatsd_url`<br>
+**Default**: `null`<br>
+The URL used to negotiate connection to DogStatsD; takes precedence over `DD_AGENT_HOST` and `DD_DOGSTATSD_PORT`. Supports `udp://` or `unix://` schemas only.
+
+`DD_DOGSTATSD_PORT`
+: **INI**: `datadog.dogstatsd_port`<br>
+**Default**: `8125`<br>
+The port used to connect to DogStatsD, will be used in combination with `DD_AGENT_HOST` to negotiate connection to DogStatsD where `DD_TRACE_HEALTH_METRICS_ENABLED` is enabled.
```
Comment (kayayarai) [inline content/en/tracing/trace_collection/library_config/php.md:175]:
```suggestion
The port used to connect to DogStatsD; used in combination with `DD_AGENT_HOST` to negotiate connection to DogStatsD where `DD_TRACE_HEALTH_METRICS_ENABLED` is enabled.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Comment by krakjoe** [comment]
@kayayarai it's good to go, thanks for the edits :)

**Review by pierotibou — COMMENTED** [review]
Shouldn't we change `dd_libraries/php.md` as well as it contains a section stating:
```
After having instrumented your application, the tracing client sends traces to localhost:8126 by default. If this is not the correct host and port change it by setting the below env variables:
````

whereas I assume the good working (This is a copy/paste from go but TBC for PHP):

```
After the application is instrumented, the trace client attempts to send traces to the Unix domain socket /var/run/datadog/apm.socket by default. If the socket does not exist, traces are sent to http://localhost:8126.
```
For .NET, we were even more precise:

```The /var/run/datadog/apm.socket Unix domain socket by default.
If the socket does not exist, then traces are sent to localhost:8126.
If a different socket, host, or port is required, use the DD_TRACE_AGENT_URL environment variable: DD_TRACE_AGENT_URL=http://custom-hostname:1234 or DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket
```

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 3]
```diff
@@ -60,11 +60,20 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 {{< partial name="apm/apm-containers.html" >}}
 </br>
 
-3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
+3. After having instrumented your application, the tracing client sends traces to Unix domain socket `/var/run/datadog/apm.socket` by default. If the socket does not exist, traces are sent to `http://localhost:8126`.  If this is not the correct host and port change it by setting the below env variables:
 
-    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.
+    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` or `DD_TRACE_AGENT_URL`.
```
Comment (pierotibou) [inline content/en/tracing/trace_collection/dd_libraries/php.md:65]: NItPicking - I think we should advertize `DD_TRACE_AGENT_URL` mainly (so maybe putting it first, even removing the others?)

File: content/en/tracing/trace_collection/dd_libraries/php.md [hunk 4]
```diff
@@ -60,11 +60,20 @@ Install and configure the Datadog Agent to receive traces from your now instrume
...
 
     See [environment variable configuration](#environment-variable-configuration) for more information on how to set these variables.
+
+Similarly, where `DD_TRACE_HEALTH_METRICS_ENABLED` is set true, the trace client attempts to send stats to the `/var/run/datadog/dsd.socket` Unix domain socket. If the socket does not exist then stats are sent to `http://localhost:8125`.
```
Comment (pierotibou) [inline content/en/tracing/trace_collection/dd_libraries/php.md:69]: `DD_TRACE_HEALTH_METRICS_ENABLED`. Interesting, I didn't know that feature. What is it about? 
It's not documented in the config page, should we document it there as well then?

**Review by pierotibou — APPROVED** [review]

File: content/en/tracing/trace_collection/dd_libraries/php.md (hunk 3 above)
Comment (krakjoe) [reply content/en/tracing/trace_collection/dd_libraries/php.md]: done

File: content/en/tracing/trace_collection/dd_libraries/php.md (hunk 4 above)
Comment (krakjoe) [reply content/en/tracing/trace_collection/dd_libraries/php.md:65]: documented

**Review by kayayarai — APPROVED** [review]

**Pull request merged**